drivers/display/display_ssd135x.c

```diff
@@ -63,116 +63,121 @@ struct ssd1351_config {
 	if (err < 0) {
 		return err;
 	}
-	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD1351_SET_VCOMH,
+	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD135X_SET_VCOMH,
 				     &config->vcomh_voltage, 1);
 	if (err < 0) {
 		return err;
 	}
-	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD1351_SET_CURRENT_ATT,
+	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD135X_SET_CURRENT_ATT,
 				     &config->current_att, 1);
 	if (err < 0) {
 		return err;
 	}
-	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD1351_SET_PRECHARGE_V,
+	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD135X_SET_PRECHARGE_V,
 				     &config->precharge_voltage, 1);
 	if (err < 0) {
 		return err;
 	}
-	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD1351_SET_PRECHARGE_P,
+	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD135X_SET_PRECHARGE_P,
 				     &config->precharge_time, 1);
 	if (err < 0) {
 		return err;
 	}
 
-	return mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD1351_STOP_SCROLL,
+	return mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD135X_STOP_SCROLL,
 				      NULL, 0);
 }
 
-static int ssd1351_resume(const struct device *dev)
+static int ssd135x_resume(const struct device *dev)
 {
-	const struct ssd1351_config *config = dev->config;
+	const struct ssd135x_config *config = dev->config;
 	int err;
 
-	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD1351_DISPLAY_ON,
+	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD135X_DISPLAY_ON,
 				     NULL, 0);
 	if (err < 0) {
 		return err;
 	}
 	return mipi_dbi_release(config->mipi_dev, &config->dbi_config);
 }
 
-static int ssd1351_suspend(const struct device *dev)
+static int ssd135x_suspend(const struct device *dev)
 {
-	const struct ssd1351_config *config = dev->config;
+	const struct ssd135x_config *config = dev->config;
 	int err;
 
-	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD1351_DISPLAY_OFF,
+	err = mipi_dbi_command_write(config->mipi_dev, &config->dbi_config, SSD135X_DISPLAY_OFF,
 				     NULL, 0);
 	if (err < 0) {
 		return err;
 	}
 	return mipi_dbi_release(config->mipi_dev, &config->dbi_config);
 }
 
-static int ssd1351_write(const struct device *dev, const uint16_t x, const uint16_t y,
+static int ssd135x_write(const struct device *dev, const uint16_t x, const uint16_t y,
 			 const struct display_buffer_descriptor *desc, const void *buf)
 {
-	const struct ssd1351_config *config = dev->config;
+	const struct ssd135x_config *config = dev->config;
 	int err;
 	size_t buf_len;
 	struct display_buffer_descriptor mipi_desc = *desc;
-	uint8_t x_position[] = {x, x + desc->width - 1};
+	uint8_t x_position[] = {x + config->column_offset,
+		x + desc->width - 1 + config->column_offset};
```

Comment:
not sure this computation doesn't risk overflowing the uint8_t?

Reply:
I see, i'll add a mask then.